pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -102,7 +102,7 @@ def test_stringified_slice_with_tz(self):
         # GH#2658
         import datetime
         start = datetime.datetime.now()
-        idx = date_range(start=start, freq="1d", periods=10)
+        idx = date_range(start=start, freq="1d", periods=10, tz='US/Eastern')
```

Comment:
Judging by the original issue, this should have been timezone aware in the first place